README.md

```diff
@@ -105,6 +105,7 @@ example, what kinds of implementation mistakes might this case help us find?
 
 There are also some convention about `expected` that you must follow:
 
+  - all keys should follow [lowerCamelCase](http://wiki.c2.com/?LowerCamelCase) convention.
```

Comment:
Looks good. 👍 

Minor grammar issues:
**A**ll keys should follow **the** [lowerCamelCase](http://wiki.c2.com/?LowerCamelCase) convention.

The following `if`s lacking capital `I`s is a separate unrelated grammatical issue that this PR need not address.

Reply:
Changes made as requested.